R/test.R

```diff
@@ -0,0 +1,4 @@
+test <- function(src = "print(sessionInfo())"){
+  r <- file.path(system.file(package="littler"), "bin/r")
```

Comment:
With what we have in `src/Makevars.in`, should this be more generic than just `r`?  I appreciate the email note by @s-u reminding us that there is only OS X binary too.  So maybe I could simplify `src/Makevars.in`?   OTOH we may also get some brave person submitting build fixes for Windoze just how it happened for RInside a few years ago (and the two packages share a fair amount of structure around embedding R).